src/components/views/directory/NetworkDropdown.js

```diff
@@ -199,6 +210,7 @@ export default class NetworkDropdown extends React.Component {
             </div>;
             current_value = <input type="text" className="mx_NetworkDropdown_networkoption"
                 ref={this.collectInputTextBox} onKeyUp={this.onInputKeyUp}
+                placeholder="matrix.org"
```

Comment:
It's just used here as an example of the name of an HS: hopefully comment clarifies.